docs/planning/sprintplanning.md

```diff
@@ -0,0 +1,66 @@
+# Knative Eventing sprint planning
+
+This document summarizes project planning mechanics for Knative Eventing releases. Knative Eventing release is planned every 6 weeks and will continue to be so. Current proposal is to align sprint with the release cycle of 6 weeks. Eventually this may change as we learn from the process.
+
+## How to manage Bugs?
+- Submit new bugs by creating a new issue of type “Report Bug”.
+- Please fill in as much detail as possible in the bug template and propose priority and severity
+- All new bugs will be labelled “triage/needstriage” and kind/bug as part of the template
+- Bugs will be triaged weekly (this could be modified based on incoming bugs rate). And upon triage will be labeled triage/accepted, or triage/rejected (and closed) with appropriate priority (priority/*) and severity (severity/*)
```

Comment:
```suggestion
- Bugs will be triaged weekly (this could be modified based on incoming bugs rate). And upon triage will be labeled triage/accepted, or triage/rejected (and closed) with appropriate priority (`priority/*`) and severity (`severity/*`)
```

Backticks will keep the `*` from becoming italics.